cypress/e2e/dataElements/Edit.spec.ts

```diff
@@ -0,0 +1,88 @@
+describe('Data elements / Edit', () => {
+    it('should change a value', () => {
+        const now = Date.now()
+
+        cy.visit('/')
+
+        // Open data elements group in side nav
+        cy.get('[data-test="sidenav"] button:contains("Data elements")', {
+            timeout: 10000,
+        }).click()
+
+        // Navigate to data element list view
+        cy.get('[data-test="sidenav"] a:contains("Data element")')
+            .first() // the selector will also grab "Data element group" and "Data element group set"
+            .click()
+
+        // Go to Edit form
+        cy.get(
+            '[data-test="dhis2-uicore-datatablecell"]:contains("ANC 1st visit")'
```

Comment:
Would it make sense to put these in constants above the test, in case the database or ordering somehow changes? Or are we then expected to update the tests?

Reply:
Yeah good point. In this case the test is not specific to "ANC 1st visit", it's just editing the first item.
I can change the selector so it does just that, regardless of the label